config/initializers/carrierwave.rb

```diff
@@ -0,0 +1,20 @@
+CarrierWave.configure do |config|
+  # 開発環境用でひとまずminioベースの設定
+  config.fog_provider = 'fog/aws'
+  config.fog_credentials = {
+    provider: 'AWS',
+    aws_access_key_id: ENV['MINIO_ROOT_USER'],
+    aws_secret_access_key: ENV['MINIO_ROOT_PASSWORD'],
+    region: ENV['MINIO_REGION'],
+    endpoint: ENV['MINIO_ENDPOINT'],
```

Comment:
環境変数の命名ですが、この部分はfogの設定であってminioもaws s3も選択肢の一つ(GCPなども使える)なので下記のようにキーと一致しているのがよいかと思います。

```suggestion
    aws_access_key_id: ENV['FOG_AWS_ACCESS_KEY_ID'],
    aws_secret_access_key: ENV['FOG_AWS_SECRET_ACCESS_KEY'],
    region: ENV['FOG_REGION'],
    endpoint: ENV['FOG_ENDPOINT'],
```

(endpointにaws s3の代わりにs3と互換性のあるminioを設定してfogを騙して動作させる)

Reply:
MYSQLの件でもご指摘いただいていたのに、使い回しを考慮できていませんでした…。
もう一点ご指摘いただいている環境変数の件と合わせて修正させていただきます！